app/views/content/life-as-a-teacher/biology/_article.html.erb

```diff
@@ -0,0 +1,103 @@
+      <li>coordination and control</li>
+      <li>photosynthesis</li>
+      <li>ecosystems</li>
+      <li>evolution, inheritance and variation</li>
+    </ul>
+    <p><a href="https://www.thenational.academy/teachers/curriculum/science-secondary/units">Explore what teaching a key stage 4 biology lesson would be like</a>.</p>
+    <p>As a secondary school biology teacher, you'll teach pupils aged 11 to 18. This means you'll always be expanding your knowledge and approach to meet pupils' needs. Once you have qualified teacher status (QTS) you can also teach any other subject, as long as you can show you have enough expertise.</p>
+  </section>
+
+  <section class="col col-space-m col-720">
+    <h2 class="heading--box-blue">Start your journey into teaching biology</h2>
+    <h3>Check your qualifications</h3>
+    <p>To train to teach in secondary schools in England, you’ll need:</p>
+    <ul>
+      <li>GCSEs at grade 4 (C) or above in English and maths</li>
+      <li>a bachelor’s degree in any subject</li>
+    </ul>
+    <p>Having relevant A levels can show your subject knowledge, if you do not have a degree in biology.</p>
+    <p>Biology teacher training courses have had successful applications from candidates with a range of degrees, including biomedical sciences, zoology, psychology, and sport and exercise sciences.
+    <p>Find out more about what <a href="/train-to-be-a-teacher/qualifications-you-need-to-teach">qualifications you need to train to teach</a>.</p>
+    <%= render Content::ExpanderComponent.new(
+      title: "check your qualifications",
+      text: "If you have qualifications from outside the UK, you'll need to show that they meet the standards set for teacher training in England.",
+      link_title: "You can get help comparing English and international qualifications.",
+      link_url: "/non-uk-teachers/non-uk-qualifications",
+    ) %>
+  </section>
+
+  <section class="col col-space-m col-720">
+    <h3>Fund your teacher training</h3>
+    <p><a href="/funding-and-support/scholarships-and-bursaries">Tax-free bursaries of <%= v :bursaries_postgraduate_biology %> are available for eligible trainee biology teachers</p>
+    <p>You can get a bursary alongside a tuition fee loan and maintenance loan.</p>
+    <p>You may also be able to get <a href="/funding-and-support">extra funding and support</a> if you're:</p>
+    <ul>
+      <li>a parent or carer</li>
+      <li>disabled</li>
+    </ul>
+    <%= render Content::ExpanderComponent.new(
+      title: "funding your training",
+      text: "If you do not have <a href=\"https://www.gov.uk/guidance/indefinite-leave-to-remain-in-the-uk\">indefinite leave to remain in the UK</a>, you're unlikely to be eligible for funding.",
+      link_title: "Find out more about fees and financial support for non-UK trainee teachers.",
+      link_url: "/non-uk-teachers/fees-and-funding-for-non-uk-trainees",
+    ) %>
+  </section>
```

Comment:
Chemistry page has section on support as an ECT - shouldn't that be here as well?

Reply:
It should! Must have been missed off the template. Have added.